service/src/main/java/uk/gov/mca/beacons/api/legacybeacon/domain/LegacyData.java

```diff
@@ -21,10 +21,14 @@ public class LegacyData implements ValueObject, Serializable {
   private LegacyEmergencyContact emergencyContact;
 
   public LegacyUse getMainUse() {
-    return getUses()
```

Comment:
This should explicitly be null if there isn't one, not an empty one. 

Although wrong - in future it's usually better to deal with an entire case i.e:

```

if (uses == null) {
return x;
}

return something;
```